### PR TITLE
#BroughtBack unstable mutation toxin to green slimes *READY TO MERGE*

### LINKED ISSUE #6866: Unstable Mutation Toxin only changes people into lizards.
When a person is injected with unstable mutation toxin, it changes them into a lizard, instead of a random race. If they are already a lizard, it just does nothing.

### PR DESCRIPTION
**Pull request by Jujumatic**
Fixes #6866

Since being able to transform into any race randomly is way more fun than transforming into a BORING ligger, I've brought back unstable mutation toxin to the best of my ability. Now 1u will transform you twice after several delayed ticks, because this edition of the toxin is EXTRA unstable! It will give you a tiny bit of meth when transforming you so that you won't get horrendously stunlocked, and will metabolise quickly enough so that it should be very difficult if not impossible to get addicted to or OD on meth. It replaces the green slime radium reaction so you should start getting this instead of stupid boring ligger mutation toxin.

:cl:
add: Unstable mutation toxin is back, and so are its recipes! Now you can turn yourself into different races chaotically and randomly like before, or turn everyone into stupid liggers if you so wish. This can be obtained by injecting radium into green slime extracts.
del: Removed lizard mutation toxin from green slime extracts because it requires radium, which overrides the original reaction that gave unstable toxin, and you can make it easily after obtaining unstable mutation toxin anyway.
/:cl:

### CODE REVIEW

**Comment by Jujumatic** [comment]
I find it strange how some line always gets deleted randomly - anyway this should be ready for reviewing

**Comment by Jujumatic** [comment]
I'll bring back the unstable mutation recipes back as well if people ree at me, but for now it should be good as it is

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm [hunk 1]
```diff
@@ -66,9 +66,6 @@
 	name = "Lizard Mutation Toxin"
 	id = "lizardmuttoxin"
 	results = list("lizardmutationtoxin" = 1)
-	required_reagents = list("radium" = 1)
```
Comment (JohnGinnane) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: how are people supposed to get this reaction now?

**Review by JohnGinnane — DISMISSED** [review]
did you test this?

**Comment by Jujumatic** [comment]
You can still get it from strange seeds - if I'm going to bring back the original recipe for lizard mutation toxin I'll have to bring in the recipes to make all the mutation toxins using unstable mutation toxin.

The only way you'd be able to get the lizard toxin is via a recipe using the unstable toxin so yeah

**Comment by Jujumatic** [comment]
Still want changes on this?

**Comment by JohnGinnane** [comment]
not sure, would maybe prefer if someone who knew more about these chems had a look

**Comment by Jujumatic** [comment]
i added all the mutation toxin recipes back, come take a look!

**Comment by Jujumatic** [comment]
this is ready to merge

**Comment by JohnGinnane** [comment]
can we get @HippieStation/head input and a @HippieStation/maintainers to look at this, I don't know enough about it myself

**Comment by Carbonhell** [comment]
i don't get it, do you prefer RNG mutations rather than this or what

**Comment by Carbonhell** [comment]
If it's rng over clear recipes, i don't like it,current system seems fine and has no trouble

**Comment by Jujumatic** [comment]
Ok so unstable mutation toxin is an added recipe, which used to be what green slimes produced if you injected radium in them. it got replaced with lizard mutation toxin so i've undone that and brought back unstable mutation toxin instead. on top of that, i've added all the unstable mutation toxin recipes that let you make all the other mutation toxins if you want. that's how the old system was, and i liked it so i thought it would be cool to bring it back. the current way we have things, there's no way to get unstable mutation toxin and no way to make mutation toxins out of it, as a result

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm (hunk 1 above)
Comment (Carbonhell) [reply code/modules/reagents/chemistry/recipes/slime_extracts.dm]: can't you modularize this

**Comment by Carbonhell** [comment]
okay,since it only adds a new recipe,sure,+1

**Comment by Jujumatic** [comment]
this should be correctly modularised now

**Comment by Pyko1** [comment]
sure, why not. +1

**Comment by GuyonBroadway** [comment]
Four words.

Did you test this?

**Comment by Jujumatic** [comment]
Tested, works fine

**Review by JohnGinnane — APPROVED** [review]
gots the votes, got the tests

**Pull request merged**